src/server/plugins/engine/plugin.ts

```diff
@@ -77,21 +86,25 @@ export const plugin = {
   dependencies: '@hapi/vision',
   multiple: true,
   register: (server: Server, options: PluginOptions) => {
-    const { modelOptions, configs, previewMode } = options
-    server.app.forms = {}
-    const forms = server.app.forms
```

Comment:
Ah this `const forms` is used a few times further down still

The routes `'/publish'`, `'/published/{id}'` and `'/published'` try to use `forms[id]`

Reply:
Resolved with 6760ebe607166cd42998a2fc12ce545a0ad9b634